src/mcp/mcpConfig.ts

```diff
@@ -16,6 +12,8 @@ export function getMCPConfigFromVSCodeSettings(
   retrieveMCPConfiguration: () => vscode.WorkspaceConfiguration = (): vscode.WorkspaceConfiguration =>
     vscode.workspace.getConfiguration('mdb.mcp'),
 ): Partial<UserConfig> {
+  const configurableProperties = new Set(Object.keys(UserConfigSchema.shape));
```

Comment:
Keys of a map are essentially a set so I don't think we're gaining much out of this `new Set` bit here.